cmd/stayrtr/stayrtr.go

```diff
@@ -49,17 +49,17 @@ var (
 	MetricsAddr = flag.String("metrics.addr", ":9847", "Metrics address")
 	MetricsPath = flag.String("metrics.path", "/metrics", "Metrics path")
 
-	ExportPath = flag.String("export.path", "/rpki.json", "Export path")
-
-	RTRVersion = flag.Int("protocol", 1, "RTR protocol version. Default is version 1 (RFC 8210)")
-	RefreshRTR = flag.Int("rtr.refresh", 3600, "Refresh interval")
-	RetryRTR   = flag.Int("rtr.retry", 600, "Retry interval")
-	ExpireRTR  = flag.Int("rtr.expire", 7200, "Expire interval")
-	SendNotifs = flag.Bool("notifications", true, "Send notifications to clients (disable with -notifications=false)")
-	EnforceVersion  = flag.Bool("enforce.version", false, "Disable version negotiation")
-	DisableBGPSec	= flag.Bool("disable.bgpsec", false, "Disable sending out BGPSEC Router Keys")
-	EnableNODELAY	= flag.Bool("enable.nodelay", false, "Force enable TCP NODELAY (Likely increases CPU)")
-
+	ExportPath           = flag.String("export.path", "/rpki.json", "Export path")
+	EnableUpdateEndpoint = flag.Bool("update.endpoint", false, "Enable HTTP endpoint that expedites the next fetch")
+
+	RTRVersion     = flag.Int("protocol", 1, "RTR protocol version. Default is version 1 (RFC 8210)")
```

Comment:
Perhaps we should add gofmt in ci/cd to check that merged code is formatted already